sites/public/lib/helpers.tsx

```diff
@@ -216,7 +216,7 @@ export const getListings = (listings) => {
     <ListingCard
       key={index}
       imageCardProps={{
-        imageUrl: imageUrlFromListing(listing, parseInt(process.env.listingPhotoSize || "1302")),
+        imageUrl: imageUrlFromListing(listing, parseInt(process.env.listingPhotoSize || "1302"))[0],
```

Comment:
For all of these calls, if we update `listingPhotoSize` in `next.config.js` to ensure that it's a number, we don't have to call parseInt in all these places or specify the 1302 fallback since it's already there.

Reply:
`listingPhotoSize: Number(process.env.LISTING_PHOTO_SIZE) || 1302,`

@seanmalbert I made the change above in next.config.js and updated the related calls. It runs fine locally but fails to build since it still registers process.env.listingPhotoSize as a string. Is there an additional step to fully implement next.config.js changes?

Reply:
@ColinBuyck ,
I looked into this a little further and nextjs doesn't do a great job of supporting typing environment variables. We could use a library like env-var to load and ensure the types are correct, but I feel like that would be its own issue. So, sorry for the back and forth, but I think we'll keep with what was there for now.

Reply:
No worries! Just reversed it and it's now building!